js/liquid.js

```diff
@@ -1008,10 +1010,11 @@ module.exports = class liquid extends Exchange {
         let network = this.safeStringUpper (params, 'network'); // this line allows the user to specify either ERC20 or ETH
         network = this.safeString (networks, network, network); // handle ERC20>ETH alias
         if (network !== undefined) {
-            request['network'] = network;
+            request['crypto_withdrawal']['network'] = network;
             params = this.omit (params, 'network');
         }
-        const response = await this.privatePostCryptoWithdrawals (this.extend (request, params));
+        request['crypto_withdrawal'] = this.extend (request['crypto_withdrawal'], params);
+        const response = await this.privatePostCryptoWithdrawals (request);
```

Comment:
We could probably use `deepExtend` here:

```JavaScript
        request['crypto_withdrawal'] = this.extend (request['crypto_withdrawal'], params);
        const response = await this.privatePostCryptoWithdrawals (request);
```

↓

```JavaScript
        const response = await this.privatePostCryptoWithdrawals (this.deepExtend (request, params));
```

In general, we do no introspection in the request format when accepting `params` and extending the request with `params`-overrides, so to keep it in line with the other exchanges the user will have to add the `crypto_withdrawal` subdict to his params-overrides.

Reply:
yeap, I haven't reminded about deepExtend's existence. also, your argument is absolute. merge that way 👍

Reply:
.